rioting-daisies/BangazonSite
# homepage #5 ticket

# Description

The default homepage now shows the 20 most recent products listed descending by date. 



Fixes Issue #5 

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. git fetch and checkout "homepage" 
1. navigate to the home page, products will be listed in descending order. If all dates are the same, you can edit date data in azure to confirm. 



# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [x ] I have added test instructions that prove my fix is effective or that my feature works
